float-features.lisp

```diff
@@ -261,26 +263,61 @@
 
 (declaim (ftype (function (T) (unsigned-byte 16)) short-float-bits))
 (defun short-float-bits (float)
-  #-mezzano
-  (declare (ignore float))
+  (declare (ignorable float))
   #+mezzano
   (mezzano.extensions:short-float-to-ieee-binary16 float)
-  #-mezzano
-  (error "Implementation not supported."))
+  #+(or ecl sbcl cmucl allegro ccl
+        (and 64-bit lispworks))
```

Comment:
Any reason 32-bit lispworks is excluded here?

Reply:
see comments in short-float-bits.
32bit lw has separate short-float and single-float types, and i was too lazy to make it generic for that and clisp :)

Reply:
Ah. weird that they would have that only on 32 bit then!